Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-alpine
```

Comment:
Would it make sense to move this to the `eng/` directory? Or better here in the root?

Reply:
After giving it more thought, keeping the `Dockerfile` in the root makes sense, alongside the `build.ps1` script. To keep the root from getting too cluttered, we _could_ move the shell script into `eng` though, as was done with the build _module_ that is found there.